pages/10.contacts/03.manage-segments/docs.en.md

```diff
@@ -82,6 +82,7 @@ A few notes for text filters:
 * `starts with`, `ends with`, `contains` filters were added later than the `like` one, so you can easily specify what you need now.
 * `%` character in the middle of the string has no special meaning. `contains` filter with `my % string` will search for a string with `%` in the middle. The same is true for `like` filter with `%my % string%` value. There is no need for escaping this character.
 * Mautic searches for `%` character in a value for `like` filter and no modification is performed if at least 1 `%` is found.
+* Using `contains` or `not contains` operator on text type fields also offer you the ability to paste a list of values from clipboard you've been copying from Spreadsheet column (Microsoft Excel and Google Spreadsheet).
```

Comment:
I think it should be 'does not contain' rather than 'not contains' grammatically speaking? Also Vale is hinting that we should not need to use parentheses here.

```suggestion
* Using `contains` or `does not contain` operators on text type fields also offer you the ability to paste a list of values from clipboard you've been copying from a spreadsheet column such as Microsoft Excel and Google Sheets.
```